README.md

```diff
@@ -36,23 +38,72 @@ To build the code without running the tests, add to the "clean build" command th
 > ./gradlew clean build -x test
 ```
 
-### Running pip tests
-Pip tests must run inside a clean pip-environment. Create a virtual environment and provide the path to its '/bin' directory using the 'BITESTS_ARTIFACTORY_PIP_ENV' variable.
+### Testing
+To run *all* tests:   
+```
+> ./gradlew clean test
+```
+
+#### Extractor tests
+```
+> ./gradlew clean build-info-api:test build-info-client:test build-info-extractor:test build-info-vcs:test
+```
+
+#### Maven tests
+```
+> ./gradlew clean build-info-extractor-maven3:test
+```
+
+#### Gradle tests
+```
+> ./gradlew clean build-info-extractor-gradle:test
+```
+
+#### Npm tests
+* Add npm executable to the system search path (PATH environment variable).
+```
+> ./gradlew clean build-info-extractor-npm:test
+```
+
+#### Go tests
+* Add Go executable to the system search path (PATH environment variable).
+```
+> ./gradlew clean build-info-extractor-go:test
+```
+
+#### Pip tests
+* Add Python and pip executables to the system search path (PATH environment variable).
+* Pip tests must run inside a clean pip-environment. Create a virtual environment and provide its path using the 'BITESTS_ARTIFACTORY_PIP_ENV' variable.  
+When running on a Windows machine, provide the path to the 'Scripts' directory.  
+When running on a unix machine, provide the path to the 'bin' directory.
+```
+> python -m venv buildinfo-tests-env
+> export BITESTS_ARTIFACTORY_PIP_ENV=/Users/user/buildinfo-tests-env/bin
+> ./gradlew clean build-info-extractor-pip:test
+```
+
+#### NuGet tests
```

Comment:
Notice that some of the section titles in the README have every word start with a capital letter, and some don't. Let's make sure we're consistent.

Reply:
Fixed.